resources/terminal

```diff
@@ -36,6 +36,28 @@ function open_iterm () {
 EOF
 }
 
+function open_iterm_new () {
```

Comment:
Can you fix the indentation of your code? I believe the standard used is 2-spaces while you have 4?